UVa/BasicRemains.java

```diff
@@ -0,0 +1,41 @@
+
+/*
+Given a base b and two non-negative base b integers
```

Comment:
I fixed the comments and used javadoc. Actually, I intentionally used that type of commenting in the beginning, so that people who are struggling to solve these problems can find these solutions a lot easier by searching with the problem descriptions. Adding asterisks or slashes in the beginning of lines can make it difficult for other people to find these solutions by simply using the problem description as a Google search query.